src/main/scala/io/iohk/ethereum/blockchain/sync/fast/FastSync.scala

```diff
@@ -560,17 +588,23 @@ class FastSync(
       syncState = syncState.updateNextBlockToValidate(header, K, X)
     }
 
-    private def handleRewind(header: BlockHeader, peer: Peer, N: Int, duration: FiniteDuration): Unit = {
+    private def handleRewind(
+        header: BlockHeader,
+        peer: Peer,
+        N: Int,
+        duration: FiniteDuration,
+        process: Boolean = true
```

Comment:
`process` parameter name is a bit confusing. What would best describe the purpose of it? Maybe continueSync or concludeSync?

Reply:
Yeah `continueSyncing` should cover it.